test/steps/dockerfile_provider.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
Should this script go to the test/framework package instead?

Reply:
well, that is a good question :)
I wasn't sure where it belongs as the line between steps and framework package is a bit blurry. But the file doesn't contain any code directly related to step, so I will move it.